test/tools/unified-spec-runner/unifed-utils.test.ts

```diff
@@ -0,0 +1,339 @@
+import { expect } from 'chai';
+
+import { mergeKMSProviders } from './unified-utils';
+
+describe('parseOptions', function () {
```

Comment:
I feel like a good spot of this is in `unit/test/tools` perhaps. Then we can leave the mocha script the same as it was. Just a preference to keep less directories we have to track for where tests are.